python/lsst/task/base/__init__.py

```diff
@@ -1,6 +1,7 @@
 from .version import *
 from .struct import *
 from .task import *
+from .task_metadata import *
+from .task_utils import *
```

Comment:
Are these utilities for internal use only? Do they need to be public?

Reply:
This module is a part of pipe/base/argumentParser.py, that supports ArgumentParser's "--show tasks" option.
This means _getTaskDict_ and _showTaskHierarchy_ functions should be public, right?

Reply:
argument parser is gen2 so that's going to go away (but not until early next year).